modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -196,14 +198,27 @@ enum VideoWriterProperties {
   VIDEOWRITER_PROP_NSTRIPES = 3,   //!< Number of stripes for parallel encoding. -1 for auto detection.
   VIDEOWRITER_PROP_IS_COLOR = 4,   //!< If it is not zero, the encoder will expect and encode color frames, otherwise it
                                    //!< will work with grayscale frames.
-  VIDEOWRITER_PROP_DEPTH = 5       //!< Defaults to CV_8U.
+  VIDEOWRITER_PROP_DEPTH = 5,      //!< Defaults to CV_8U.
+  VIDEOWRITER_PROP_HW_ACCELERATION = CAP_PROP_HW_ACCELERATION, //!< Set/get hardware acceleration type (value is enum cv::VideoAccelerationType). Setting supported only via `params` parameter in VideoCapture constructor. Default is VIDEO_ACCELERATION_NONE.
+  VIDEOWRITER_PROP_HW_DEVICE       = CAP_PROP_HW_DEVICE  //!< Hardware device index (select GPU if multiple available)
 };
 
 //! @} videoio_flags_base
 
 //! @addtogroup videoio_flags_others
 //! @{
 
+//! Video Acceleration type (used as value in CAP_PROP_HW_ACCELERATION and VIDEOWRITER_PROP_HW_ACCELERATION)
+// (in case of FFMPEG backend, it translated to enum https://github.com/FFmpeg/FFmpeg/blob/master/libavutil/hwcontext.h#L27)
```

Comment:
> /master/

Probably we should have fixed commit / tag to keep point on the right place
(or just remove the line number and add enum name `AVHWDeviceType`).

Reply:
Ok!